src/stuf/templatetags/stuf.py

```diff
@@ -0,0 +1,92 @@
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from typing import Any, Literal
+
+from django.template import Library
+from django.utils import dateformat, timezone
+
+from ..constants import EndpointSecurity
+from ..models import StufService
+
+register = Library()
+
+
+DATE_FORMAT = "%Y%m%d"
+TIME_FORMAT = "%H%M%S"
+DATETIME_FORMAT = "%Y%m%d%H%M%S"
+
+
+def fmt_soap_datetime(d: datetime) -> str:
+    return d.strftime(DATETIME_FORMAT)
+
+
+def fmt_soap_date(d: datetime) -> str:
+    return d.strftime(DATE_FORMAT)
+
+
+def fmt_soap_time(d: datetime) -> str:
+    return d.strftime(TIME_FORMAT)
+
+
+@register.inclusion_tag("stuf/includes/security.xml")
+def render_security(service: StufService, expiry_minutes: int) -> dict[str, Any]:
+    """
+    Provide the security headers block based on service configuration.
+    """
+    now = timezone.now()
+    expires_at = now + timedelta(minutes=expiry_minutes)
+    return {
+        "use_wss": (
+            service.soap_service.endpoint_security
+            in [EndpointSecurity.wss, EndpointSecurity.wss_basicauth]
+        ),
+        "wss_username": service.soap_service.user,
+        "wss_password": service.soap_service.password,
+        # uh - fmt as date seems wrong? what's the point of specifying an expiry in
+        # minutes if you truncate the time part??? This is taken from the StUF-ZDS
+        # implementation, note that StUF-BG does not format at all!
+        "wss_created": fmt_soap_date(now),
+        "wss_expires": fmt_soap_date(expires_at),
```

Comment:
this just looks plain wrong, why calculate things with second/minute precision to then truncate the time part?

Taken from the StUF-ZDS implementation - the StUF-BG implementation doesn't even format it at all but just dumps the datetime instance into the template context, and in the templates there was no formatting either (just a plain `{{ created }}`)

Reply:
checked with spec (link via Joeri, https://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-soap-message-security-1.0.pdf), they need to be ISO 8601 datetimes

Reply:
Fixed via 95f3e1497c39fb59bf4c05dac1d726f8b0cb36d3